plugins/notify.py

```diff
@@ -15,26 +15,28 @@ def send_notification(self, build_id, text):
             for room in notification_list:
                 self.say("{} BuildID: {}, Message: {}".format(' '.join('@'+user for user in notification_list.get(room, [])), build_id, text), room=self.get_room_from_name_or_id(room), notify=True) 
 
-    @respond_to("^subscribe (@?)(?P<user>\S+) to (?P<build_id>\S+)")
-    def subscribe(self, message, user, build_id):
+    @respond_to("^subscribe (@?)(?P<users>(\S+ )+)to(?P<build_ids>( \S+)+)")
+    def subscribe(self, message, users, build_ids):
         """
-        subscribe [user] to [buildid]: request to be notified when builds complete
+        subscribe [user, user, ...] to [buildid, buildid, ...]: request to be notified when builds complete
```

Comment:
This makes it seem like the users should be comma separated instead of space separated.